docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -537,11 +545,16 @@ We charge applicable sales tax in US jurisdictions as required.
 
 **How do I cancel my account?**
 
-To cancel your Temporal Cloud account, [contact Support](https://temporalsupport.zendesk.com/).
+If you used our self sign-up feature to create your Temporal Account, and you are an Account Owner, you can cancel your account from the Temporal Cloud [Billing](https://cloud.temporal.io/billing) page under the Plan tab.
+You need appropriate administrative permissions to access this section.
+Select the "Delete Account" button.
+Otherwise, to cancel your Temporal Cloud account, please [contact Support](https://temporalsupport.zendesk.com/).
```

Comment:
Should this line be `to **delete** your Temporal Cloud account`... As it feels like `Delete` & `Cancel` are 2 different options when they are same ?

Reply:
Changing to delete as suggested.